From 04baf9967a84adb6856fdb69fda80553719a4bfa Mon Sep 17 00:00:00 2001
From: Michael Forney <mforney@mforney.org>
Date: Sun, 26 Jun 2016 12:11:13 -0700
Subject: [PATCH] scanner: Use int rather than uint for loop index

uint requires including sys/types.h and defining _GNU_SOURCE. We could use
unsigned instead, but there is no reason not to use just int instead.
---
 src/scanner.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/scanner.c b/src/scanner.c
index 5f06e8e..836a181 100644
--- a/src/scanner.c
+++ b/src/scanner.c
@@ -808,7 +808,7 @@ find_enumeration(struct protocol *protocol,
 	struct interface *i;
 	struct enumeration *e;
 	char *enum_name;
-	uint idx = 0, j;
+	int idx = 0, j;
 
 	for (j = 0; j + 1 < strlen(enum_attribute); j++) {
 		if (enum_attribute[j] == '.') {
-- 
2.9.0

